README.rst

```diff
@@ -1,36 +1,40 @@
+deployment of nuclear energy systems in service to civilization and the environment.
 
 More specifically, ARMI:
 
 * Provides a hub-and-spoke mechanism to standardize communication and coupling between
   physics kernels and the specialist analysts who use them,
-* Facilitates the creation and execution of detailed models and complex analysis methodologies,
-* Provides an ecosystem within which to rapidly and collaboratively build new analysis and physics
-  simulation capabilities, and
+
+* Facilitates the creation and execution of detailed models and complex analysis
+  methodologies,
+
+* Provides an ecosystem within which to rapidly and collaboratively build new analysis
+  and physics simulation capabilities, and
+
 * Provides useful utilities to assist in reactor development.
 
-Using ARMI plus a collection of ARMI-aware physics plugins, an engineering team can perform a
-full analysis of a reactor system and then repeat the same level of analysis with some changed
-input parameters for almost no additional cost. Even better, thousands of perturbed cases can be
-executed in parallel on large computers, helping conceptual design teams home in on an optimal
-design, or helping detailed design teams understand sensitivities all the way from, for example,
-an impurity in a control material to the peak structural temperature in a design-basis transient.
+Using ARMI plus a collection of ARMI-aware physics plugins, an engineering team can
+perform a full analysis of a reactor system and then repeat the same level of analysis
+with some changed input parameters for almost no additional cost. Even better, thousands
+of perturbed cases can be executed in parallel on large computers, helping conceptual
+design teams home in on an optimal design, or helping detailed design teams understand
+sensitivities all the way from, for example, an impurity in a control material to the
+peak structural temperature in a design-basis transient.
 
-.. note:: ARMI does not come with a full selection of physics kernels. They will need to be
-      acquired or developed for your specific project in order to make full use of this tool.
-      Many of the example use-cases discussed in this manual require functionality that is 
-      not included in the open-source ARMI Framework.
+.. note:: ARMI does not come with a full selection of physics kernels. They will need to
+   be acquired or developed for your specific project in order to make full use of this
+   tool.  Many of the example use-cases discussed in this manual require functionality
+   that is not included in the open-source ARMI Framework.
 
 In general, ARMI aims to enhance the quality, ease, and rigor of computational nuclear
-reactor design and analysis. Additional high-level overview about this system can be found in
-[#touranarmi]_.
+reactor design and analysis. Additional high-level overview about this system can be
+found in [#touranarmi]_.
 
 
 .. list-table:: Quick links
```

Comment:
can you update the coming soon thing here too?

Reply:
Done